samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/ShadowContainerSamplePage.WinUI.xaml

```diff
@@ -310,7 +310,51 @@
 							<StackPanel Spacing="8">
 								<TextBlock Text="Complex Shadow Shapes" Style="{StaticResource TitleTextBlockStyle}" />
 
-								<StackPanel>
+										<!--
+									Should be the same as: https://developer.mozilla.org/fr/docs/Web/CSS/CSS_backgrounds_and_borders/Box-shadow_generator
```

Comment:
how can they be the same when the order of shadows are reversed?
unless the shadows are "commutative" which i dont think is the case...